column_transforms/rasgo_concat/rasgo_concat.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+*
+{%- for col_expr, vals in new_cols %}
```

Comment:
i think this one can be simplified by taking one `column_list` as the arg and concatenating them in order that they are supplied

so ['col1', 'col2', 'col3'] would generate a new column col1 || col2 || col3 as ConcatenatedCol